### ISSUE TITLE
Add support for vitest 3 

### ISSUE THREAD

**Issue by Smrtnyk** [vitest-community/vitest-browser-vue#6]
I am getting this when using vitest 3 beta

```text
 WARN  Issues with peer dependencies found
packages/frontend
└─┬ vitest-browser-vue 0.0.1
  ├── ✕ unmet peer @vitest/browser@^2.1.0-beta.4: found 3.0.0-beta.2
  └── ✕ unmet peer vitest@^2.1.0-beta.4: found 3.0.0-beta.2

```

**Comment by sheremet-va** [comment]
Fixed in 8177ae9

**Comment by badsgahhl** [comment]
@sheremet-va could you release a new version to npm for vitest 3 support?
